source/connect/favorite-connections/import-export-ui/export.txt

```diff
@@ -0,0 +1,119 @@
+.. _compass-export-connections-ui:
+
+==============================================
+Export Favorite Connections on |compass-short|
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can export favorite connections on |compass|. Colleagues can import your 
+favorite connections, or you can use them in your other workspaces.
+
+About This Task
+~~~~~~~~~~~~~~~
+
+When you export favorite connections, |compass-short| exports the list
+of connections as a JSON file.
+
+By default, when you export favorite connections, database usernames and
+passwords are included in plaintext. To protect sensitive connection
+information, you can omit secrets from the file or encrypt the exported file 
+with a passphrase.
```

Comment:
Can we single-source this copy with the "About This Task" section on the CLI page?